invenio/ext/elasticsearch/config/es_config.py

```diff
@@ -0,0 +1,162 @@
+# Configure the elasticsearch mappings for each type
```

Comment:
@glouppe this was already discussed. @marioskogias should go back to solution that was using elasticsearch section in JSONAlchemy configuration. You can also check #2587 that brings new native search engine implementation. We should try to agree on new search API (pagination, facets/aggregations, sorting/ranking).